src/utils/deploy/hash-fns.js

```diff
@@ -107,7 +107,13 @@ const hashFns = async (
     normalizedPath: path.basename(functionPath, path.extname(functionPath)),
     runtime,
   }))
+  const functionSchedules = functionZips.reduce((acc, { name, schedule }) => {
```

Comment:
[dust] could this be map + filter instead? this usage of `reduce` makes it hard to skim for me.

Reply:
Done in https://github.com/netlify/cli/pull/3885/commits/d2cfbb8467edbce2c1cdc1fe28b08007bf81ce74.